main.go

```diff
@@ -115,6 +115,9 @@ var Config struct {
 
 	// Client Certificate expiration time
 	ClientCertExpirationTime uint `envconfig:"CLIENT_CERT_EXPIRATION_DAYS" default:"30"`
+
+	// MaxConcurrentReconciles is the maximum number of concurrent Reconciles which can be run
+	MaxConcurrentReconciles uint `envconfig:"MAX_CONCURRENT_RECONCILES" default:"3"`
```

Comment:
Question to learn: why is `MaxConcurrentReconciles` the same for edgedevice_controller, edgedeployment_controller and edgedevicelabels_controller?

Reply:
we can tune it as we go, if we find out a need for a different granularity for each.
the first idea was to process the same amount of requests as with 3 replicas.

Reply:
@masayag any reason?